tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.cs

```diff
@@ -276,9 +290,24 @@ public void TestBinaryOperations()
             Assert.Equal(false, df.GreaterThan(listOfInts)[5, 0]);
             Assert.Equal(false, df.LessThan(5)[5, 0]);
             Assert.Equal(false, df.LessThan(listOfInts)[5, 0]);
-
-            // The original DF is untouched
-            Assert.Equal(0, df[0, 0]);
+            // The following binary ops are in place
+            Assert.Equal(5, df.Add(5, true)[0, 0]);
```

Comment:
A nice, readable way of coding with booleans is to use named parameters:

```C#
Assert.Equal(5, df.Add(5, inPlace: true)[0, 0]);
```

It makes it really easy for a new person reading the code understand what `true` means here.